persistence/src/main/scala/hmda/persistence/processing/ProcessingMessages.scala

```diff
@@ -25,5 +25,6 @@ object ProcessingMessages {
   case class ParsingCompletedWithErrors(submissionId: SubmissionId) extends Event
   case class ValidationCompleted(submissionId: SubmissionId) extends Event
   case class ValidationCompletedWithErrors(submissionId: SubmissionId) extends Event
+  case class Persisted() extends Event
```

Comment:
Case classes need to have parenthesis by default, this didn't use to be the case but with current `Scala` versions it is mandatory. If you try to do something like this

```scala
case class Person
```

You will get a compiler error as follows:

```shell
error: case classes without a parameter list are not allowed;
use either case objects or case classes with an explicit `()' as a parameter list.
```
Case objects (singletons), on the other hand, do not need parenthesis. I modeled this as a case class initially so that some additional data could be passed back to the sender (i.e. the time when something was persisted). Since I ended up not using that, I'm going to revert this to a `case object`